src/test/resources/org/prebid/server/it/test-application.properties

```diff
@@ -350,3 +350,4 @@ gdpr.default-value=1
 gdpr.vendorlist.v1.cache-dir=src/test/resources/org/prebid/server/it/gdpr-vendorlist1
 gdpr.vendorlist.v2.cache-dir=src/test/resources/org/prebid/server/it/gdpr-vendorlist2
 ccpa.enforce=false
+price-floors.enabled=true
```

Comment:
I would like to use this property with true value, only for `FloorsTest`. With this property tests will fail for other adapters